it/v3/endpoints/CreateAndAmendOtherExpensesControllerISpec.scala

```diff
@@ -29,15 +29,14 @@ import shared.support.IntegrationBaseSpec
 class CreateAndAmendOtherExpensesControllerISpec extends IntegrationBaseSpec {
 
   "Calling the create and amend endpoint" should {
-    "return a 200 status code" when {
+    "return a 204 status code" when {
       "any valid request is made" in new NonTysTest {
         override def setupStubs(): Unit = {
           DownstreamStub.onSuccess(DownstreamStub.PUT, downstreamUri, NO_CONTENT, JsObject.empty)
         }
 
         val response: WSResponse = await(request().put(requestBodyJson))
-        response.status shouldBe OK
-        response.json shouldBe responseBody
```

Comment:
this can be replaced to verify that there is no body
response.body.isEmpty shouldBe true